modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -193,12 +194,30 @@
   /** The key to look for in the service configuration file to override the {@link DEFAULT_INGEST_ZIP_JOB_LOAD} */
   public static final String ZIP_JOB_LOAD_KEY = "job.load.ingest.zip";
 
+  /** The source to download from  */
+  public static final String DOWNLOAD_SOURCE = "org.opencastproject.download.source";
+
+  /** The user for download from external sources */
+  public static final String DOWNLOAD_USER = "org.opencastproject.download.user";
+
+  /** The password for download from external sources */
+  public static final String DOWNLOAD_PASSWORD = "org.opencastproject.download.password";
+
   /** The approximate load placed on the system by ingesting a file */
   private float ingestFileJobLoad = DEFAULT_INGEST_FILE_JOB_LOAD;
 
   /** The approximate load placed on the system by ingesting a zip file */
   private float ingestZipJobLoad = DEFAULT_INGEST_ZIP_JOB_LOAD;
 
+  /** The user for download from external sources */
+  private static String downloadUser = DOWNLOAD_USER;
```

Comment:
@CGreweling Looks like you wanted to set a default value for downloadUser, downloadPassword and downloadSource but accidentally set the names of the respective configuration properties ;-)